ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/rules/JDBCAbstractSplitFilterRule.java

```diff
@@ -0,0 +1,178 @@
+package org.apache.hadoop.hive.ql.optimizer.calcite.rules;
```

Comment:
Can we move JDBC rules to ```org.apache.hadoop.hive.ql.optimizer.calcite.rules.jdbc``` package?

In addition, we are missing the license header here. You can run ```mvn clean apache-rat:check``` to make sure all files contain the appropriate header.